javascript/javascript1/week1/dogage.js

```diff
@@ -0,0 +1,13 @@
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture-dogYearOfBirth;
+
+let shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears == true) 
```

Comment:
it is better to get used to using `===` not `==` to compare the both the value and type

Reply:
done